requirements.txt

```diff
@@ -11,4 +11,6 @@ django-environ==0.4.5
 social-auth-app-django==3.1.0
 social-auth-core==3.2.0
 pytest==5.3.1
-pytest-django==3.5.1
\ No newline at end of file
+pytest-django==3.5.1
+google-cloud-pubsub==1.1.0
```

Comment:
I think for now it is totally fine to load google-pubsub and kafka packages at the same time, but for the future we could maybe think of another way of solving this. Like for instance adding extra python packages in extra herbie-repositories like one herbie-pubsub-adapter and one herbie-kafka-adapter. What do you think?

Reply:
This is a really nice idea. We sure have to do something like this in the close future